src/users/store/reducers/users.reducer.ts

```diff
@@ -10,6 +10,7 @@ export interface UsersListState {
   reinvitePendingUser: User;
   editUserFailure: boolean;
   userDetails: User;
+  accessTypes: Array <String>;
```

Comment:
The accessTypes array is not type String, it needs to model the data returned from the api.
may need to create a new model to handle this that is based on the accestypes returned